src/test/regress/output/multi_outer_join.source

```diff
@@ -732,6 +732,64 @@ LOG:  join order: [ "multi_outer_join_right" ][ local partition join "multi_oute
         20
 (20 rows)
 
+-- flattened out subqueries with outer joins are also supported
+SELECT
+  l1.l_custkey,
+  count(*) as cnt
+FROM (
+  SELECT l_custkey, l_nationkey
+  FROM multi_outer_join_left
+  WHERE l_comment like '%a%'
+) l1
+LEFT JOIN (
+  SELECT r_custkey, r_name
+  FROM multi_outer_join_right
+  WHERE r_comment like '%b%'
+) l2 ON l1.l_custkey = l2.r_custkey
+GROUP BY l1.l_custkey
+ORDER BY cnt DESC, l1.l_custkey DESC
+LIMIT 20;
+LOG:  join order: [ "multi_outer_join_left" ][ local partition join "multi_outer_join_right" ]
+ l_custkey | cnt 
+-----------+-----
+        27 |   1
+        24 |   1
+        23 |   1
+        22 |   1
+        21 |   1
+        15 |   1
+        13 |   1
+        12 |   1
+        11 |   1
+(9 rows)
+
+-- this query is flattened out to following by postgresql
+SELECT 
+	l1.l_custkey, count(*) AS cnt
+FROM (
+	multi_outer_join_left l1 
+	LEFT JOIN
+	multi_outer_join_right l2
+	ON ((l1.l_custkey = l2.r_custkey))) 
+WHERE ( l1.l_comment like '%a%' AND l2.r_comment like '%b%') 
```

Comment:
Shouldn't the l2.r_comment like '%b%' be in the ON() section? Otherwise this would be transformed into an inner join.